docs/development/hacking_itk_wasm.md

```diff
@@ -1,69 +1,252 @@
 # Hacking ITK-Wasm
 
-Thank you for contributing a pull request!
+## TLDR
 
-**Welcome to the ITK community!**
+Install Docker, and
+
+```sh
+npm i -g pnpm
+pnpm install
+pnpm build
+pnpm test
+```
+
+and contribute the patch with standard GitHub best practices.
+
+## Introduction
+
+**Welcome to the ITK community! ☀️**
 
 We are glad you are here and appreciate your contribution. Please keep in mind our [community participation guidelines](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CODE_OF_CONDUCT.md).
 
-## Build dependencies
+Thank you in advance for contributing a pull request! 🙏
+
+## High level overview
+
+The following sections describe how to contribute to [ITK-Wasm's constituent parts](../introduction/parts.md).
+
+We follow [standard GitHub contribution best practices]: pull requests made from forks are tested with continuous integration tests, and updates are made via code review before integration.
+
+We use the [conventional commit] standard for our commit message format in the *itk-wasm* repository.
 
-- [Node.js / NPM](https://nodejs.org/en/download/)
-- [Docker](https://docs.docker.com/install/)
-  * On Linux, make sure you can run [`docker` without `sudo`](https://askubuntu.com/questions/477551/how-can-i-use-docker-without-sudo).
-  * On Windows, we recommend [WSL 2 with Docker enabled](https://docs.docker.com/desktop/windows/wsl/).
-- Bash
+The C++ core and Docker build environment can be developed independently from the CLI, language-specific libraries, and example packages. The latter are developed as a [pnpm workspace]. Individual packages in the workspace can be developed independently after `pnpm install` is executed at the root.
 
-## Building
+## C++ core
+
+ITK-Wasm's [C++ core](../introduction/parts.md#cxx-core) can be developed with native toolchains. The steps are:
+
+0. Install a [C++ compiler toolchain] and [CMake]
+1. Build [ITK]
+2. Build the `WebAssemblyInterface` module from this module against ITK
+3. Run the tests
+
+From a command line, these steps in brief are:
 
-To build itk-wasm itself:
 ```sh
-npm install
-npm run build
+# Build ITK
+git clone https://github.com/InsightSoftwareConsortium/ITK
```

Comment:
Shouldn't this be build with our forked version of ITK?
https://github.com/InsightSoftwareConsortium/itk-wasm/blob/main/src/docker/itk-wasm-base/Dockerfile#L40

Reply:
Great point! For developing the `WebAssemblyInterface` code, either will work, but I will make a note about where to make changes to ITK.